tests/acl_loader_test.py

```diff
@@ -82,6 +83,17 @@ def test_vlan_id_not_a_number(self, acl_loader):
             acl_loader.rules_info = {}
             acl_loader.load_rules_from_file(os.path.join(test_path, 'acl_input/illegal_vlan_nan.json'))
 
+    def test_ethertype_translation(self, acl_loader):
+        acl_loader.rules_info = {}
+        acl_loader.load_rules_from_file(os.path.join(test_path, 'acl_input/acl1.json'))
+        assert acl_loader.rules_info[("DATAACL", "RULE_3")]
+        assert acl_loader.rules_info[("DATAACL", "RULE_3")] == {
+            "VLAN_ID": 369,
+            "ETHER_TYPE": 35020,
```

Comment:
from yang model translation, it is a integer, but our default rule is a string. it is ok as redis db is always string based, but there is a slight difference here